### ISSUE TITLE
conditionally loading the Rollbar serviceprovider does not work with L.5.5's package auto discovery

### ISSUE THREAD

**Issue by cenob8** [rollbar/rollbar-php-laravel#30]
Hi,
Using Laravel 5.5's auto package discovery will always load the Rollbar service provider. So if you want to conditionally load the Rollbar service for a specific environment as suggested in the docs this will not work.
Solution : add the Rollbar package to the dont-discover list in composer.json
Perhaps this should be added to the docs.

```javascript
"extra": {
        "laravel": {
            "dont-discover": [
                "rollbar/rollbar-laravel"
            ]
        }
    },
```

**Comment by drbyte** [comment]
Remember: even if this package is "discovered", it will do nothing if there are no `ROLLBAR_XXXX` entries in your `.env` .... so, since you control each environment with your `.env` content, you also control whether the Rollbar package will do any reporting.

**Comment by ArturMoczulski** [comment]
@drbyte Thanks for helping with this. It's definitely a way to achieve conditional loading. I don't want users to rely on existance of `.env` credentials to support it though. While it achieves the same thing, I believe using `dont-discover` and manual conditional loading is much more clear. I modified the conditional loading instructions: https://github.com/rollbar/rollbar-php-laravel/blob/master/README.md#enabling--disabling-rollbar-on-specific-environments

@cenob8 I added instructions to the doc right here: https://github.com/rollbar/rollbar-php-laravel/blob/master/README.md#disabling-laravel-55s-auto-discovery
